react/Alerter/Readme.md

```diff
@@ -3,11 +3,18 @@ Include the `<Alert />` component somewhere in your app, but only once. After th
 
 ```
 import Alerter from 'cozy-ui/transpiled/react/Alerter';
-// we have to put the Alert only once in the app
+
+if (isTesting()) {
+  setTimeout(() => {
+    Alerter.success('This is a success alert')
+  }, 1)
```

Comment:
Since we've CSS transition (opacity) I think we'll have a few false positives doing this.